api/libproject/devstart.rb

```diff
@@ -1190,7 +1190,7 @@ def fix_desynchronized_billing_project_owners(cmd_name, *args)
   fc_config = get_fc_config(op.opts.project)
   domain = get_config(op.opts.project)["googleDirectoryService"]["gSuiteDomain"]
   flags = ([
-      ["--fc-base-url", fc_config["baseUrl"]],
+      ["--rawls-base-url", fc_config["rawlsBaseUrl"]],
```

Comment:
Those are tools that require this parameter. A quicker for doing code review is to search for 
`rawls-base-url` and you will find two changed files. This one and FetchWorkspaceDetails.
Other tools did not require this paramter, instead of pulling this config. Our tools are a bit messy and I suspect most of them are not working actually.